.lighthouserc.json

```diff
@@ -1,15 +1,21 @@
 {
   "ci": {
     "collect": {
-      "numberOfRuns": 1,
+      "numberOfRuns": 2,
```

Comment:
Why is this changed?

Reply:
Lighthouse performance scores will change due to inherent variability in web and network technologies, even if there hasn't been a code change. When we implement Github actions we should run Lighthouse multiple times and beware of variability before drawing conclusions about a performance-impacting change. You can see here in the lighthouse-ci-documentation https://github.com/GoogleChrome/lighthouse/blob/v6.0.0-beta.0/docs/variability.md

Reply:
Understood but still, I wouldn't modify it in this PR.

Reply:
Got it I'll change it

Reply:
Done